config/default.ts

```diff
@@ -11,6 +11,7 @@ export default {
   },
   server: {
     addressRequestLimit: 50,
+    transactionRequestLimit: 100,
```

Comment:
I tried making mock transactions and I could go to about ~500 inputs before the tx was too big, so maybe let's just make this 500